compile.c

```diff
@@ -543,6 +543,31 @@ verify_list(ISEQ_ARG_DECLARE const char *info, LINK_ANCHOR *const anchor)
 #define verify_list(info, anchor) verify_list(iseq, (info), (anchor))
 #endif
 
+static void
+verify_call_cache(rb_iseq_t *iseq)
+{
+    return; /* comment out to enable */
```

Comment:
It seems `#if CPDEBUG` is used in this file to do the same thing.

Reply:
Thanks! I went for this instead of CPDEBUG since CPDEBUG doesn't build at the moment. In retrospect, that doesn't really matter since I need to do manual edit to enable this debug code with the current setup anyway.

Anyways, I have a fix for CPDEBUG at #2586.

Reply:
Great, thanks for the fix.